runtime/compiler/trj9/x/env/J9CPU.cpp

```diff
@@ -24,71 +24,85 @@
 #include "env/CompilerEnv.hpp"
 #include "env/CPU.hpp"
 #include "env/VMJ9.h"
+#include "x/runtime/X86Runtime.hpp"
 
 namespace J9
 {
 
 namespace X86
 {
 
-const char *
-CPU::getX86ProcessorVendorId(TR::Compilation *comp)
+TR_X86CPUIDBuffer *
+CPU::queryX86TargetCPUID()
    {
-   return comp->fej9()->getX86ProcessorVendorId();
+   static TR_X86CPUIDBuffer buf = { {'U','n','k','n','o','w','n','B','r','a','n','d'} };
+   jitGetCPUID(&buf);
+   return &buf;
    }
 
-uint32_t
-CPU::getX86ProcessorSignature(TR::Compilation *comp)
+const char *
+CPU::getX86ProcessorVendorId()
    {
-   return comp->fej9()->getX86ProcessorSignature();
+   static char buf[13];
+
+   // Terminate the vendor ID with NULL before returning.
+   //
+   strncpy(buf, self()->queryX86TargetCPUID()->_vendorId, 12);
+   buf[12] = '\0';
+
+   return buf;
    }
 
 uint32_t
-CPU::getX86ProcessorFeatureFlags(TR::Compilation *comp)
+CPU::getX86ProcessorSignature()
    {
-   return comp->fej9()->getX86ProcessorFeatureFlags();
+   return self()->queryX86TargetCPUID()->_processorSignature;
```

Comment:
same "duplicate of OMR" comment here

Reply:
^ that was my guess what the problem was, I could be completely wrong

Reply:
Just did another set of tests, so "my duplicates" was not able to get rid of this error `[ERR] JVMJ9VM011W Unable to load j9jit29: /tmp/bld_386111/j2sdk-image/jre/lib/amd64/compressedrefs/libj9jit29.so: undefined symbol: _ZN2TR9JitConfig8instanceEv`.   `_ZN2TR9JitConfig8instanceEv` is something in OMRCPU's `queryX86TargetCPUID()`.

Reply:
I will get rid of the duplicates, these did not resolve the problem.

Reply:
I think removing the J9 specific macro in OMRCPU's queryX86TargetCPUID() is causing this undefined symbol problem (_ZN2TR9JitConfig8instanceEv).